services/weather.js

```diff
@@ -60,11 +56,11 @@ module.exports.insertWeatherDataByCity = async (weatherData) =>
 {
     try
     {
-        console.log(`service insert weather data`);
-        return Weather.create(weatherData);
+        const insertWeatherResponse = await Weather.create(weatherData);
+        return insertWeatherResponse;
     } catch (error)
     {
-        return error
+        throw new BadRequestError;
```

Comment:
we might need a message here so we can know where the BadRequestError is from

Reply:
same to other places

Reply:
yup, that is true :D I added it there